cmd/team.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"io"
+
+	team "github.com/astronomer/astro-cli/team"
+	"github.com/spf13/cobra"
+)
+
+func newTeamCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "team",
+		Short: "Manage Astronomer Teams",
+		Long:  "Teams represents a Team in or a group from an IDP in the Astronomer Platform",
```

Comment:
`a Team in or`, did you mean `A team or a group from an IDP in the Astronomer Platform` ?

Reply:
typo, ill fix that, thanks! second suggestion is correct.